Tutorial/walkthroughs/GoogleAnalytics/base_workflow/dags/sample_ga.py

```diff
@@ -0,0 +1,116 @@
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.operators.dummy_operator import DummyOperator
+from plugins.google_analytics_plugin.operators.google_analytics_reporting_to_s3_operator import GoogleAnalyticsReportingToS3Operator
+
+
+s3_bucket = 'astronomer-workflows-dev'
```

Comment:
somewhat styling preference but when I'm defining variables at the top I like to make them all caps

so `s3_bucket --> S3_BUCKET`